### PR TITLE
feat(cdp): one-click action migration

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

We need to make it easy for people to migrate from action webhooks to CDP destinations

## How did you test this code?

- Test for migration
- Clicked around in the UI

### CODE REVIEW

**Comment by mariusandra** [comment]
I made cohort handling more robust.



Still missing: an error if you _add_ a cohort

File: frontend/src/lib/api.ts [hunk 1]
```diff
@@ -1009,6 +1009,9 @@ const api = {
                 .withQueryString(temporaryToken ? `temporary_token=${temporaryToken}` : '')
                 .update({ data: actionData })
         },
+        async migrate(id: PluginConfigTypeNew['id']): Promise<HogFunctionType> {
```
Comment (benjackwhite) [inline frontend/src/lib/api.ts:1012]: Why plugin config?

**Review by benjackwhite — APPROVED** [review]
Looks good to me. One tiny nit that I think is just a copy pasta issue.

**Comment by mariusandra** [comment]
These snapshots are getting annoying. Anyway, I added one last red message when you add a cohort filter to an action that didn't have it before:



The "will be disabled" part isn't enforced, and I'm afraid that might be a deeper rabbit hole, so going to leave as is for now. I already started refactoring the actions page logics, but quickly backtracked...

**Pull request merged**